src/linagora.esn.unifiedinbox/app/components/inbox-configuration/configuration.less

```diff
@@ -4,3 +4,7 @@
 @import './forwardings/form/inbox-forwardings-form';
 @import './forwardings/user/inbox-forwardings-user';
 @import './read-receipt/request-receipts/request-receipts';
+
+inbox-configuration .esn-tab > li a {
```

Comment:
`esn-tab` is a esn component so you can fix it directly here : https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/blob/main/src/frontend/css/modules/tabs.less

Reply:
This one may impact the other modules, so we need this only in inbox module